src/scmdata/database.py

```diff
@@ -54,10 +57,12 @@ def _check_is_subdir(root, d):
 
 def _get_safe_filename(inp):
     def safe_char(c):
-        if c.isalnum() or c in "-/*_.":
+        if c.isalnum() or c in f"-{os.sep}*_.":
```

Comment:
Hmm i'm not sure how this will handle replacing a \. Not sure off the top of my head if python will assume that the \ is an escape character (https://www.w3schools.com/python/gloss_python_escape_characters.asp).

Reply:
Ye I'm pretty sure that solution doesn't work, I tried another in an update...